javascript/exercises/cashRegisterSolution.js

```diff
@@ -0,0 +1,49 @@
+var prices = {
+  brenda: { 
+    banana: "1.25",
+    handkerchief: ".99",
+    Tshirt: "25.01"
+    },
+  claire: {
+    "apple": "0.60",
+    "nalgene": "10.34",
+    "protein_shake": "22.36"
+    }
+};
+
+
+function cashRegister(cart){
+  var people = Object.keys(cart); // array of the carts keys, which are people  
+  var items = []; // array of the each people keys, which are items  
+  
+  for (var i = 0; i < people.length; i++) {
+    // for each person grab the every item
+    items[i] = Object.keys(prices[people[i]]);    
```

Comment:
The thing about `items[i] = Object.keys(prices[people[i]]);` is that it allows the people and items arrays to have identical index number. So people[0] has in their cart items found at items[0] (items is an array with arrays in every index).
`_items.push(prices[people[i]])_` doesn't allow for this parallel trend.